hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestMetadataTableWithSparkDataSource.scala

```diff
@@ -84,23 +84,21 @@ class TestMetadataTableWithSparkDataSource extends SparkClientFunctionalTestHarn
       .mode(SaveMode.Append)
       .save(basePath)
 
-    // Files partition of MT
-    val filesPartitionDF = spark.read.format(hudi).load(s"$basePath/.hoodie/metadata/files")
+    val mdtDf = spark.read.format("hudi").load(s"$basePath/.hoodie/metadata")
+    mdtDf.show()
```

Comment:
what the purpose of the `show` calling?

Reply:
yeah let's remove this.. can be helpful for debugging but already our logs are bloated.